README.md

```diff
@@ -172,11 +172,18 @@ There are 5 different environment tiles, of which some have different styles of
 #### Locations
 As of January 2022, there are a total of 29 different locations in game.
 
-A lot of the earlier details such as what order which locations were unlocked or released in are no longer easily found without restarting the game, however looking at the game's [version history][history], the rough suggested order is:
+A lot of the earlier details such as what order which locations were unlocked or released in are no longer easily found without restarting the game, however looking at the game's [version history][history], the suggested order is:
+
+1. Prague & Warsaw ([1.3](https://www.ipa4fun.com/ipa/64334/))
+2. UK & Scandinavia ([1.4](https://www.ipa4fun.com/ipa/68834/))
+3. Bernina ([1.6](https://www.ipa4fun.com/ipa/81402/))
+4. Belfast ([1.8](https://www.ipa4fun.com/ipa/92847/))
+5. Mostar ([1.10](https://www.ipa4fun.com/ipa/106999/))
+6. Madrid ([1.12](https://www.ipa4fun.com/ipa/124229/))
+7. Budapest ([1.14](https://www.ipa4fun.com/ipa/267007/))
```

Comment:
```suggestion
7. Budapest ([1.14.1](https://www.ipa4fun.com/ipa/267007/))
```

Reply:
I'm not sure if `ipa4fun` hasn't stored `v1.14`, or if it never existed, but in either case it's probably more accurate to put this since we have a source.